Server/src/BaseQuasarServer.cpp

```diff
@@ -335,27 +339,77 @@ int BaseQuasarServer::parseCommandLine(
+            *isHelpOrVersionOrExtra = true;
         return 0;
     }
+
     //Print version if needed
     if (printVersion)
     {
-    	std::cout << VERSION_STR << std::endl << \
-    			"\t BuildHost: " << BuildMetaInfo::getBuildHost() << std::endl << \
-				"\t BuildTimestamp: " << BuildMetaInfo::getBuildTime() << std::endl << \
-				"\t CommitID: " << BuildMetaInfo::getCommitID() << std::endl << \
-				"\t ToolkitLibs: " << BuildMetaInfo::getToolkitLibs() << std::endl;
-        if (isHelpOrVersion)
-            *isHelpOrVersion = true;
+        #ifdef BACKEND_UATOOLKIT
+        std::string uasdkCoreModuleVersionInfo = VersionInfoCoreModule::getCoreModuleVersionInfo().toUtf8();
+        #else
+        std::string uasdkCoreModuleVersionInfo = "N/A";
+        #endif
+
+        std::cout << VERSION_STR << std::endl << \
+                Quasar::TermColors::ForeBold() << "\t BuildHost: " << Quasar::TermColors::StyleReset() << BuildMetaInfo::getBuildHost() << std::endl << \
+                Quasar::TermColors::ForeBold() << "\t BuildTimestamp: " << Quasar::TermColors::StyleReset() << BuildMetaInfo::getBuildTime() << std::endl << \
+                Quasar::TermColors::ForeBold() << "\t CommitID: " << Quasar::TermColors::StyleReset() << BuildMetaInfo::getCommitID() << std::endl << \
+                Quasar::TermColors::ForeBold() << "\t ToolkitLibs: " << Quasar::TermColors::StyleReset() << BuildMetaInfo::getToolkitLibs() << std::endl << \
+                Quasar::TermColors::ForeBold() << "\t BuildTooliktPath: " << Quasar::TermColors::StyleReset() << OpcuaToolkitInfo::getOpcuaToolkitPath() << std::endl;
+
+        if (isHelpOrVersionOrExtra)
+            *isHelpOrVersionOrExtra = true;
+        return 0;
+    }
+    else if (printExtraInfo)
+    {
+        auto stringWithoutSlashes = [](std::string& s) {
+                                std::replace( s.begin(), s.end(), '\\', ' ') ;
+                                return s;
+                            };
+
+        #ifdef BACKEND_UATOOLKIT
+        std::string uasdkCoreModuleVersionInfo = VersionInfoCoreModule::getCoreModuleVersionInfo().toUtf8();
+        std::string uaStackVersionInfo = VersionInfoCoreModule::getUaStackVersionInfo().toUtf8();
+        std::string uaStackVersion = VersionInfoCoreModule::getUaStackVersion().toUtf8();
+        std::string uaStackStaticConfigInfo = VersionInfoCoreModule::getUaStackStaticConfigInfo().toUtf8();
+        std::string uaStackRuntimeConfigInfo = VersionInfoCoreModule::getUaStackRuntimeConfigInfo().toUtf8();
+        std::string uaStackPlatformLayerVersionInfo = VersionInfoCoreModule::getUaStackPlatformLayerVersionInfo().toUtf8();
+        std::string uaStackPlatformLayerVersion = VersionInfoCoreModule::getUaStackPlatformLayerVersion().toUtf8();
+        std::string uaStackPlatformLayerConfigInfo = VersionInfoCoreModule::getUaStackPlatformLayerVersion().toUtf8();
+        #else
+        std::string uasdkCoreModuleVersionInfo = "N/A";
```

Comment:
Suggest to say "N/A in open62541 backend" or so to be more clear why the information is not there.